orchestrator/orchestrator.go

```diff
@@ -119,21 +128,24 @@ func (s *Orchestrator) execute(filter func(wf *process.Process, node process.Nod
 	}
 }
 
-func (s *Orchestrator) executeNode(wf *process.Process, n process.Node, exec *execution.Execution, event *event.Event, data map[string]interface{}) error {
-	logrus.WithField("module", "orchestrator").WithField("nodeID", n.ID()).WithField("type", fmt.Sprintf("%T", n)).Debug("process process")
-	if node, ok := n.(*process.Task); ok {
+func (s *Orchestrator) executeNode(wf *process.Process, n *process.Process_Node, exec *execution.Execution, event *event.Event, data map[string]interface{}) error {
+	logrus.WithField("module", "orchestrator").
+		WithField("nodeID", n.ID()).
+		WithField("type", fmt.Sprintf("%T", n)).Debug("process process")
+
+	if node, ok := n.Type.(*process.Process_Node_Task_); ok {
```

Comment:
We could use something like that to be more consistent and I find it easier to read but definitely not something blocking
```suggestion
	if task := n.GetTask(); task != nil {
```

Reply:
Done